src/ifconfig.interfaces.js

```diff
@@ -25,13 +27,16 @@ module.exports = function (cp) {
           return f(stderr);
         }
 
-        f(null, parse(ifConfigOut, routeOut));
+        f(null, parse(ifConfigOut, routeOut, fs.readFileSync(interfaces.FILE, {encoding: 'UTF-8'})));
```

Comment:
there seems to be a case where it would break backward compatibility if `/etc/network/interfaces` is not available or if the read throw an error

Reply:
Prints warning, when there was an error reading the interfaces file, but proceeds with the parsing of the interfaces